sql/catalyst/src/main/java/org/apache/spark/sql/connector/util/V2ExpressionSQLBuilder.java

```diff
@@ -263,4 +280,8 @@ protected String visitTrim(String direction, String[] inputs) {
       return "TRIM(" + direction + " " + inputs[1] + " FROM " + inputs[0] + ")";
     }
   }
+
+  protected String visitExtract(String[] inputs) {
+    return "EXTRACT(" + inputs[0] + " FROM " + inputs[1] + ")";
```

Comment:
the doc says EXTRACT has 3 parameters: https://github.com/apache/spark/pull/36663/files#diff-85a02a33061cf83f3f1cac24e0ba805eabb8d833905c76a4181258431d894755R219